assignment.js

```diff
@@ -31,7 +31,11 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 1:')
 
 // Add your code below this line
-
+let baseNumber = 2
+for (let i = 0; i <= 12; i++) {
+    let result = baseNumber * i
+    console.log(result)
+}
 // Add your code above this line
 
```

Comment:
@rlou88 Good Job using a for loop to complete this task. One thing is to make sure your indentation is 2 spaces(line 36 & 37), currently, you have the code indented with 4 spaces.
Take a look at this approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L34)